content/patterns/checkbox/checkbox-pattern.html

```diff
@@ -14,61 +14,62 @@
   </head>
   <body>
     <main>
-      <h1>Checkbox Pattern</h1>
+      <h1>チェックボックス パターン</h1>
+      <!-- 翻訳元リビジョン: https://github.com/w3c/aria-practices/tree/d6cf9d8db772603a456e7d7ef52e409d39124301 -->
 
       <section id="about">
-        <h2>About This Pattern</h2>
-        <p>WAI-ARIA supports two types of <a href="#checkbox" class="role-reference">checkbox</a> widgets: dual-state checkboxes toggle between two choices -- checked and not checked -- and tri-state checkboxes, which  allow an additional third state known as partially checked.</p>
+        <h2>このパターンについて</h2>
+        <p>WAI-ARIA は、2種類の <a href="#checkbox" class="role-reference">チェックボックス</a> ウィジェットをサポートしています：二つの状態のチェックボックスは、--チェック済みとチェック済みでない--の2つの選択肢を切り替えるもので、三つの状態のチェックボックスは、これに加えて「部分的にチェックされた」という第三の状態を許容するものです。</p>
         <p>
-          One common use of a tri-state checkbox can be found in software installers where a single tri-state checkbox is used to represent and control the state of an entire group of install options.
-          And, each option in the group can be individually turned on or off with a dual state checkbox.
+          三つの状態のチェックボックスの一般的な使用方法の1つは、ソフトウェアのインストーラです。三つの状態のチェックボックスがインストールオプションのグループ全体の状態を表し、制御するために使用されます。
```

Comment:
原文は1文ですね。
「インストーラです」ではなく「インストーラの中で...」かなと
たとえばこう?


```suggestion
          三状態チェックボックスの一般的な使用方法の1つは、ソフトウェアのインストーラの中で見られるもので、単一の三状態チェックボックスが、インストールオプションのグループ全体の状態を表示、制御するために使われます。
```